packages/vike-node/package.json

```diff
@@ -5,41 +5,79 @@
   "main": "./dist/index.js",
   "types": "./dist/index.d.ts",
   "exports": {
-    "./connect": "./dist/connect.js",
-    "./fastify": "./dist/fastify.js",
-    "./h3": "./dist/h3.js",
-    "./hono": "./dist/hono.js",
-    "./elysia": "./dist/elysia.js",
     "./plugin": "./dist/plugin/index.js",
     ".": "./dist/index.js",
-    "./__handler": {
-      "vike-node-dev": "./dist/runtime/handler-web-and-node.js",
-      "edge-light": "./dist/runtime/handler-web-only.js",
-      "worker": "./dist/runtime/handler-web-only.js",
-      "workerd": "./dist/runtime/handler-web-only.js",
-      "browser": "./dist/runtime/handler-web-only.js",
-      "default": "./dist/runtime/handler-web-and-node.js",
-      "types": "./dist/runtime/handler-web-only.d.ts"
+    "./handler": {
```

Comment:
My thinking is that if the user wants to use `vike-node` then it's mostly for the benefits I mentioned. Now if the user is accessing the universal handler itself then he won't get any of these benefits, right?

The way I see it is that it's basically increasing the surface area of the API (and consequently increasing the surface area for breaking changes), which is by far the most complicated thing to maintain about Open Source, for not *that* much added value. The user could recreate the universal handler himself on the user land.

Reply:
> it doesn't conceptually increase the surface area

I see, makes sense.

My only concern left is that it forgoes all the core benefits of vike-node. Another way to put it: if we would move Vike's universal handler inside Vike core then I'd agree. I feel like it's a bit weird to have the user installing `vike-node` just for using Vike's universal middleware. Actually, maybe we really should do that: move Vike's universal handler inside Vike core?

Reply:
> move Vike's universal handler inside Vike core?

Yes, it would make more sense. `vike-node` would still have its own `./handler` though (which would wrap the `vike` universal one).
Also, `./handler` being present doesn't mean we need to document it.

Reply:
As it's auto-generated by `universal-middleware`, I would need to add an option to not generate it. It's you think that's really important, I can add it. (I think it's not that important, but I think you have a stringer feeling about this than I do)

Reply:
Let's keep it then, marking as resolved.